bot/src/cardData/holidayData.ts

```diff
@@ -196,38 +196,31 @@ export const holidaysData = [
   {
     holidayName: "Good Friday",
     holidayImageUrl: "https://upload.wikimedia.org/wikipedia/commons/thumb/3/33/W%C3%BCger_Kreuzigung.jpg/360px-W%C3%BCger_Kreuzigung.jpg",
-    holidayDate: "04-15 00:00:00 GMT+0",
+    holidayDate: "04-07 00:00:00 GMT+0",
     holidayCountry: "United Kingdom",
     holidayDescription: "Good Friday was historically a common law holiday and is recognised as an official public holiday(also known as a Bank Holiday).All state schools are closed and most businesses treat it as a holiday for staff; however, many retail stores now remain open.Government services in Northern Ireland operate as normal on Good Friday, substituting Easter Tuesday for the holiday. There has traditionally been no horse racing on Good Friday in the UK.However, in 2008, betting shops and stores opened for the first time on this day and in 2014 Lingfield Park and Musselburgh staged the UK's first Good Friday race meetings. The BBC has for many years introduced its 7 am News broadcast on Radio 4 on Good Friday with a verse from Isaac Watts' hymn 'When I Survey the Wondrous Cross'."
   },
   {
     holidayName: "Easter Monday",
     holidayImageUrl: "https://upload.wikimedia.org/wikipedia/commons/thumb/9/9f/21.4.14_Lamacaes_1_Easter_Monday_Parade_070_%2813983298816%29.jpg/255px-21.4.14_Lamacaes_1_Easter_Monday_Parade_070_%2813983298816%29.jpg",
-    holidayDate: "04-18 00:00:00 GMT+0",
+    holidayDate: "04-10 00:00:00 GMT+0",
     holidayCountry: "United Kingdom",
     holidayDescription: "Easter Monday refers to the day after Easter Sunday in either the Eastern or Western Christian traditions. It is a public holiday in some countries. It is the second day of Eastertide. In Western Christianity, it marks the second day of the Octave of Easter, and in Eastern Christianity it marks the second day of Bright Week."
   },
   {
     holidayName: "Early May bank holiday",
     holidayImageUrl: "https://c.tadst.com/gfx/750w/early-may-bank-holiday.jpg",
-    holidayDate: "05-02 00:00:00 GMT+0",
+    holidayDate: "05-01 00:00:00 GMT+0",
     holidayCountry: "United Kingdom",
     holidayDescription: "May Day, officially known as Early May Day Bank Holiday, is a combination of two holidays in the UK:\n 1. May Day is an ancient celebration of spring, rebirth, and fertility.\n2. International Workers' Day, also known as Labour Day, is about workers' rights."
   },
   {
     holidayName: "Spring bank holiday",
     holidayImageUrl: "https://c.tadst.com/gfx/750w/springbankhol.jpg",
-    holidayDate: "06-02 00:00:00 GMT+0",
+    holidayDate: "05-29 00:00:00 GMT+0",
     holidayCountry: "United Kingdom",
     holidayDescription: "The spring bank holiday, also known as the late May bank holiday, is a time for people in the United Kingdom to have a day off work or school. It falls on the last Monday of May but it used to be on the Monday after Pentecost."
   },
-  {
```

Comment:
This holiday doesn't exist anymore?

Reply:
It does not exist in 2023, so delete it for now